salt/loader.py

```diff
@@ -64,6 +62,24 @@
 except ImportError:
     HAS_PKG_RESOURCES = False
 
+# Construct the string that goes inbetween the module name and the .pyc extension on
+# Python 3.5 and above.  Example: "some_module.cpython-36.pyc".  We need this when
+# several versions of Python access Salt in the same directory.  Otherwise
+# we get tons of "Bad magic number" messages because Salt under Py 3.8 will blindly
+# try to load modules that were converted to .pyc on Py 3.6.
+if sys.version_info[:2] >= (3, 5):
+    try:
+        import platform
```

Comment:
Where do we don't have the `platform` module?

Reply:
I refactored this to use `sys.implementation.cache_tag`, it's cleaner.